### PR TITLE
Automated PR for Bahyph

### PR DESCRIPTION
**Pull request by myndzi**

### CODE REVIEW

File: src/Bahyph.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+<SPDX name="Bahyph License" identifier="Bahyph" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/Bahyph</url>
+  </urls>
+  <license>
+    <optional>
+      <p>COPYRIGHT NOTICE</p>
+    </optional>
+    <body>
+      <p>These patterns and the generating sh script are Copyright (c) GMV 1991</p>
```
Comment (jlovejoy) [inline src/Bahyph.xml]: I think this should be noted with the <copyright> tag??

File: src/Bahyph.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<SPDX name="Bahyph License" identifier="Bahyph" osi-approved="false">
+  <urls>
+    <url>https://fedoraproject.org/wiki/Licensing/Bahyph</url>
+  </urls>
+  <license>
+    <optional>
+      <p>COPYRIGHT NOTICE</p>
```
Comment (jlovejoy) [inline src/Bahyph.xml]: should this be optional or part of copyright tag (see other comment)??

Comment (myndzi) [inline src/Bahyph.xml]: The copyright tag is intended to surround an actual copyright notice, so this falls more under the matching rules about omittable text

File: src/Bahyph.xml (hunk 1 above)
Comment (jlovejoy) [inline src/Bahyph.xml]: discussed on call; decided not to add copyright tag and never had matching markup, so not adding it now as more of a one-off license

**Pull request merged**